### PR TITLE
WP: Chapter 4: TODOs processed.

### PR DESCRIPTION
**Pull request by IlyaMoskva**
Issue #35 solved.

### CODE REVIEW

File: white-paper/wp.tex [hunk 1]
```diff
@@ -3,7 +3,258 @@
 \usepackage{float}
 \usepackage[bottom]{footmisc}
 \usepackage{bookmark}
-\usepackage{microtype}
+\usepackage{microtype}\documentclass[12pt,oneside]{article}
```
Comment (g4s8) [inline white-paper/wp.tex:6]: @IlyaMoskva it seems to be a duplicate: we already have `documentclass` at the beginning

File: white-paper/wp.tex [hunk 2]
```diff
@@ -3,7 +3,258 @@
...
+                          \begin{textblock}{0.125}[0,0](0.5,0)
+                            \tikz \node[fill=myblack,inner sep=0, minimum width=0.125\TPHorizModule,minimum height=16\TPVertModule] {};
+                          \end{textblock}
+                        }
+                      \usepackage{xcolor}
+                        \definecolor{firebrick}{HTML}{B22222}
+                        \definecolor{myred}{HTML}{CF0A2C}
+                        \definecolor{myblack}{HTML}{232527}
+                      \newcommand\dd[1]{\colorbox{gray!30}{\texttt{#1}}}
+                      \usepackage{hyperref}
```
Comment (g4s8) [inline white-paper/wp.tex:45]: @IlyaMoskva why so many packages added here?

**Review by g4s8 — CHANGES_REQUESTED** [review]
@IlyaMoskva did you try to build this document? It seems something is wrong with this PR

File: white-paper/wp.tex (hunk 2 above)
Comment (IlyaMoskva) [reply white-paper/wp.tex:45]: Hm.. According to data from Git, they were added by you on Oct-7 in this commit e29e033111bff160e3b4ae18647515aabadf47ec

File: white-paper/wp.tex (hunk 1 above)
Comment (IlyaMoskva) [reply white-paper/wp.tex]: @g4s8 It was copy-paste issue (I work in LaTeX online editor). I've dropped this commit and fixed again. It's strange, why build was successful.

**Comment by IlyaMoskva** [comment]
@g4s8 , please re-review changes, now the commit is OK technically, just chapter 4 contain changes.

**Review by g4s8 — APPROVED** [review]
@IlyaMoskva thanks

**Pull request merged**